test/common/http/http2/codec_impl_test.cc

```diff
@@ -3639,6 +3640,32 @@ TEST_P(Http2CodecImplTest, ConnectTest) {
   driveToCompletion();
 }
 
+// CONNECT without upgrade type gets tagged with "bytestream"
```

Comment:
Given that the codec strips the `:protocol = bytestream` header, I'm not sure I understand this comment.